logs/app.log

```diff
@@ -0,0 +1,7 @@
+2021-05-09 21:40:51,807 INFO mate.academy.service.AuthenticationServiceImpl:14 Login method was called. With login = bob
```

Comment:
please remove this file locally and make commit and push
this file should not be in the PR